x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -95,3 +98,23 @@ const LatestFindingsPageTitle = () => (
     />
   </PageTitle>
 );
+
+const getFindingsDistribution = ({
+  total,
+  passed,
+  failed,
+  currentPageSize,
+  pageIndex,
+  pageSize,
+}: Record<'currentPageSize' | 'total' | 'passed' | 'failed', number | undefined> &
+  Record<'pageIndex' | 'pageSize', number>) => {
+  if (!total || !passed || !failed || !currentPageSize) return;
+
+  return {
+    total,
+    passed,
+    failed,
+    pageStart: pageIndex * pageSize + 1,
+    pageEnd: pageIndex * pageSize + currentPageSize,
```

Comment:
this is the actual fix.

say we have `82` items 
and we show `10` per page
on our last page we will show 2 items

but the text counters used to say we're displaying `81-90` instead of `81-82` 

worth noting: our range is inclusive